apiServer.js

```diff
@@ -45,6 +45,23 @@ function humanOutput(res, json) {
   htmlOutput = null;
 }
 
+function escapeHtml(unsafe) {
+  return unsafe
+    .replace(/&/g, "&amp;")
+    .replace(/</g, "&lt;")
+    .replace(/>/g, "&gt;")
+    .replace(/"/g, "&quot;")
+    .replace(/'/g, "&#039;");
+}
+
+function safeFields(unsafe) {
```

Comment:
@MattIPv4 What are your thoughts on replacing this with:

```javascript
function safeFields(unsafe) {
  return _.map(unsafe, escapeHtml)
}
```

Reply:
Ah yeah, makes far more sense.